src/test/spec/ocsp.rs

```diff
@@ -0,0 +1,47 @@
+use std::time::Duration;
+
+use crate::{
+    options::TlsOptions,
+    test::{CLIENT_OPTIONS, LOCK},
+    Client,
+};
+use bson::doc;
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+async fn run() {
+    let _guard = LOCK.run_exclusively().await;
+
+    let should_succeed = std::env::var("OCSP_TLS_SHOULD_SUCCEED")
+        .unwrap()
+        .to_lowercase();
+
+    let mut options = CLIENT_OPTIONS.clone();
+    let mut tls_options: TlsOptions = options.tls.clone().unwrap().options().cloned().unwrap();
```

Comment:
this process is quite awkward with the current API since we have to go through `Option` -> `Tls` -> `TlsOptions`. I'm not sure what the answer is so I just added a `#[cfg(test)]` accessor and figured we could try to improve this API later.

Reply:
Is there a reason you couldn't use `ClientOptions::tls_options` (or modify it to return a reference, if you wanted to avoid the clone)? I originally added it for pretty much this exact type of use case

Reply:
nope, just didn't know that method existed. updated to use that.